lib/examples/npdsch_enodeb.c

```diff
@@ -197,12 +203,12 @@ void base_init()
   // init memory
   sf_buffer = srsran_vec_cf_malloc(sf_n_re);
   if (!sf_buffer) {
-    perror("malloc");
+    perror("Error : malloc for sf_buffer");
```

Comment:
same goes for all such cases

Reply:
OK - will ensure there is consistency as per the above